src/extension/gogoduck/doc/wps-acorn.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I'm not going to lose sleep over this.  You're right, it might be useful to have a handful of examples that work out of the box. Leave as is for now.